diff --git a/src/p4p/__init__.py b/src/p4p/__init__.py
index 99f0261..cfc8f88 100644
--- a/src/p4p/__init__.py
+++ b/src/p4p/__init__.py
@@ -1,10 +1,16 @@
 import logging
+import os
 import sys
 import atexit
 
 try:
     # give a chance to adjust DSO loader path
     import epicscorelibs.path
+    try:
+        # Fix for Python 3.8 under Windows. See: https://docs.python.org/3.8/whatsnew/3.8.html#bpo-36085-whatsnew
+        os.add_dll_directory(epicscorelibs.path.lib_path)
+    except Exception:
+        pass
 except ImportError:
     pass
 
